client/starwhale/utils/venv.py

```diff
@@ -549,15 +550,18 @@ def get_conda_prefix_path(name: str = "") -> str:
     return output.decode().strip()
 
 
+@lru_cache(maxsize=12)
```

Comment:
since the method has no parameter, `maxsize` is needless

Reply:
ok